litmus-portal/frontend/src/components/AccountsTab/AccountSettings/index.tsx

```diff
@@ -0,0 +1,239 @@
+import {
+  FormControl,
+  IconButton,
+  Input,
+  InputAdornment,
+  InputLabel,
+} from '@material-ui/core';
+import Divider from '@material-ui/core/Divider';
+import Typography from '@material-ui/core/Typography';
+import Visibility from '@material-ui/icons/Visibility';
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+import React from 'react';
+import pass from '../../../assets/icons/pass.svg';
+import PasswordModal from '../PasswordModal';
+import PersonalDetails from '../PersonalDetails';
+import useStyles from './styles';
+
+// used for password field
+interface State {
+  password: string;
+  err: boolean;
+  showPassword: boolean;
+}
+
+// AccountSettins displays the starting page of "Accounts" tab
+const AccountSettings: React.FC = () => {
+  const classes = useStyles();
+
+  // use for password validation
+  const regularExpression = /^(?=.*[0-9])(?=.*[a-zA-Z])(?=.*[!@#$%^&_*])[a-zA-Z0-9!@#$%^&_*]{8,16}$/;
```

Comment:
What is the password requirements?

Reply:
The password should contain special characters, numbers and letters, 'regularExpression is used to check whether the password contains all the three.